src/main/java/com/taboola/backstage/services/AdvertiserReportsService.java

```diff
@@ -0,0 +1,102 @@
+
+    /**
+     * <p>
+     *    The "Top Campaign Content" report is an advertiser report. This report lists the top 1000 Items of
+     *    a Campaign. The report allows fetching the top 1000 Items for all Campaigns of an Account, or
+     *    filter the results to include only the Items of a specific Campaign.
+     * </p>
+     *
+     * @param auth Authentication object {@link BackstageAuthentication}
+     * @param accountId Specific {@link com.taboola.backstage.model.Account Account.getId()}
+     * @param startDate From {@link LocalDate} period
+     * @param endDate To {@link LocalDate} period
+     * @param filters Optional report filters ({@link TopCampaignContentOptionalFilters})
+     * @return Report data in form of {@link TopCampaignContentReport} object
+     * @throws BackstageAPIUnauthorizedException {@link com.taboola.backstage.model.auth.Token Token} is expired or bad credentials
+     * @throws BackstageAPIConnectivityException Connectivity issues (HTTP status 5xx)
+     * @throws BackstageAPIRequestException Bad request (HTTP status 4xx)
+     */
+    TopCampaignContentReport getTopCampaignContentReport(BackstageAuthentication auth, String accountId,
+                                                         LocalDate startDate, LocalDate endDate,
+                                                         Map<TopCampaignContentOptionalFilters, String> filters) throws BackstageAPIUnauthorizedException, BackstageAPIConnectivityException, BackstageAPIRequestException;
+
+    /**
+     * <p>
+     *    The "Campaign Summary" report is an advertiser report. This report provides general campaign
+     *    metrics such as impressions, clicks, conversions, spend, as well as performance metrics such
+     *    as CTR, CPC, CPM, CPA. The report can be broken down by date, campaign, referring site,
+     *    country or platform.
+     * </p>
+     *
+     * @param auth Authentication object {@link BackstageAuthentication}
+     * @param accountId Specific {@link com.taboola.backstage.model.Account Account.getId()}
+     * @param startDate From {@link LocalDate} period
+     * @param endDate To {@link LocalDate} period
+     * @param dimension {@link CampaignSummaryDimensions} enum
+     * @return Report data in form of {@link CampaignSummaryReport} object
+     * @throws BackstageAPIUnauthorizedException {@link com.taboola.backstage.model.auth.Token Token} is expired or bad credentials
+     * @throws BackstageAPIConnectivityException Connectivity issues (HTTP status 5xx)
+     * @throws BackstageAPIRequestException Bad request (HTTP status 4xx)
+     */
+    CampaignSummaryReport getCampaignSummeryReport(BackstageAuthentication auth, String accountId, LocalDate startDate, LocalDate endDate, CampaignSummaryDimensions dimension) throws BackstageAPIUnauthorizedException, BackstageAPIConnectivityException, BackstageAPIRequestException;
```

Comment:
typo (Summary)

Reply:
This will cause backward issue now with 1.0.1 and 1.0.0 version but I think its worth fixing this.